src/Build/BackEnd/Components/RequestBuilder/RequestBuilder.cs

```diff
@@ -1079,6 +1080,10 @@ private void SetProjectCurrentDirectory()
 
             _projectLoggingContext = _nodeLoggingContext.LogProjectStarted(_requestEntry);
 
+            // Now that the project has started, parse a few known properties which indicate warning codes to treat as errors or messages
+            //
+            ConfigureWarningsAsErrorsAndMessages();
```

Comment:
Do you foresee any performance degradation by adding this call?  I assumed it was safe since the project has "started" building...